<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>newRepair</class>
 <widget class="QDialog" name="newRepair">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>530</width>
    <height>554</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dodawanie kosztu</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>491</width>
     <height>271</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="l_title">
      <property name="text">
       <string>Tytuł</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="l_date">
      <property name="text">
       <string>Data</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="l_cost">
      <property name="text">
       <string>Koszt</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="l_milage">
      <property name="text">
       <string>Przebieg</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="l_ctype">
      <property name="text">
       <string>Rodzaj</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="l_carid">
      <property name="text">
       <string>Samochód</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="le_title"/>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="le_milage"/>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="le_cost"/>
    </item>
    <item row="1" column="1">
     <widget class="QDateEdit" name="dateEdit"/>
    </item>
    <item row="5" column="1">
     <widget class="QComboBox" name="cb_carid"/>
    </item>
    <item row="2" column="1">
     <widget class="QComboBox" name="cb_ctype"/>
    </item>
   </layout>
  </widget>
  <widget class="QLineEdit" name="le_note">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>330</y>
     <width>441</width>
     <height>111</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Maximum">
     <horstretch>0</horstretch>
     <verstretch>30</verstretch>
    </sizepolicy>
   </property>
  </widget>
  <widget class="QLabel" name="l_note">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>350</y>
     <width>47</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Notatki</string>
   </property>
  </widget>
  <widget class="QPushButton" name="b_addnew">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>470</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Dodaj</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>le_title</tabstop>
  <tabstop>dateEdit</tabstop>
  <tabstop>cb_ctype</tabstop>
  <tabstop>le_milage</tabstop>
  <tabstop>le_cost</tabstop>
  <tabstop>cb_carid</tabstop>
  <tabstop>le_note</tabstop>
  <tabstop>b_addnew</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
